simulatingrisk/hawkdove/model.py

```diff
@@ -276,10 +276,12 @@ def step(self):
         self.datacollector.collect(self)
         if self.converged:
             self.running = False
-            print(
-                f"Stopping after {self.schedule.steps} rounds. "
-                + f"Final rolling average % hawk: {round(self.rolling_percent_hawk, 2)}"
-            )
+            # FIXME: this output is annoying in batch runs
+            # print(
+            #     f"Stopping after {self.schedule.steps} rounds. "
+            #     + "Final rolling average % hawk: "
+            #     + f"{self.rolling_percent_hawk: .2f}"
+            # )
```

Comment:
Perhaps some kind of verbosity flag could be used.

Reply:
I have a new issue (#61) to address this. I'm hoping to find an easy way to include the status (running/converged) in the interface, definitely going to add the status to the collected data so it's easy to check if the batch runs converged (since we set a max # of steps). I could clean this up now but I think it will make more sense to do as part of #61.

Reply:
That makes sense to me, I just figured I'd make a comment but expect this to be handled at a later time.